### PR TITLE
Fixed sorting orders of returned json files

### PR DESCRIPTION
**Pull request by ywkim312**
To test, please run the main.py in you local and go to 

1. http://localhost:5000/pdl/programs/conservation/eqip/state-distribution
if you see Texas as the first item, it works fine

2. http://localhost:5000/pdl/programs/conservation/csp/state-distribution
if you see South Dakota first, it works fine.

### CODE REVIEW

**Comment by pengyin-shan** [comment]
@ywkim312 , both of them have Texas as the first state. Are the links in your ticket descript correct?

**Review by pengyin-shan — APPROVED** [review]
run locally and the pr works as the description

File: app/pdl.yaml [hunk 1]
```diff
@@ -297,26 +297,6 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
-  /pdl/programs/snap/state-distribution:
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply app/pdl.yaml:300]: there were two of these so I remove the one. if you see the yaml file, there is still one of this left in there

File: app/main.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
 app.config['SQLALCHEMY_DATABASE_URI'] = db_connenction_url
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 app.config['PROPAGATE_EXCEPTIONS'] = True
+app.config['JSON_SORT_KEYS'] = False
```
Comment (sandeep-ps) [inline app/main.py:35]: Also, do you think this is needed? If so, it's not being used in the current logic. Please see.

Comment (ywkim312) [reply app/main.py:35]: Yes, it is needed. Even though you make it pdl.py using the json with correctly order, flask sorts again if this is not there.

Comment (ywkim312) [reply app/main.py:35]: You can test it in your local, removing this line. Then you will see the returned json starts from Alabama instead of texas

Comment (sandeep-ps) [reply app/main.py:35]: Oh! Okay. Thanks. I thought this was regular config. Now I realized that it's a Flask configuration.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**